### DISCUSSION TITLE
How to move a line to the up or bottom?

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#12040] [Feature Requests]
I can move a line with "⌥ + up/down" in VSCode. How to do that in Zed?

**Comment by SomeoneToIgnore** [comment]
`cmd-shift-p` opens a command panel where all commands could be searched:

![image](https://github.com/zed-industries/zed/assets/2690773/2674779b-588a-4619-ba2a-cc5ff8a46d9d)
